py/server/deephaven/dbc/__init__.py

```diff
@@ -1,5 +1,35 @@
 #
 #     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
 #
+"""The dbc package includes the modules and functions for using external databases with Deephaven."""
 
-"""The dbc package includes the modules for using external databases with Deephaven."""
+from deephaven import DHError
+from deephaven.table import Table
+import deephaven.arrow as dharrow
+
+
+def read_sql(conn: str, query: str) -> Table:
+    """Executes the provided SQL query via ConnectorX and returns a Deephaven table.
+
+    Args:
+        conn (str): a connection string URI, please refer to https://sfu-db.github.io/connector-x/databases.html for
+            database specific format
+        query (str): SQL query statement
+
+    Returns:
+        a new Table
+
+    Raises:
+        DHError
+    """
+    try:
+        import connectorx as cx
+    except ImportError:
+        raise DHError(message="import ConnectorX failed, please install it first.")
```

Comment:
Should this give the pip install command?

Reply:
Probably not, I think it is safe to assume a Python user knows?

Reply:
The message is not clear that the package is `connectorx`.  If they try `ConnectorX`, the install will fail.  As it is, this message will probably require a google, which is not ideal.